vc-verifier/kotlin/vcverifier/src/test/java/io/mosip/vercred/vcverifier/credentialverifier/validator/MsoMdocValidatorTest.kt

```diff
@@ -49,27 +51,27 @@ class MsoMdocValidatorTest {
         }
 
-        assertEquals(ERROR_MESSAGE_INVALID_DATE_MSO,verificationException.errorMessage)
-        assertEquals(ERROR_CODE_INVALID_DATE_MSO,verificationException.errorCode)
+        assertEquals(ERROR_MESSAGE_INVALID_VALID_UNTIL_MSO,verificationException.errorMessage)
+        assertEquals(ERROR_CODE_INVALID_VALID_UNTIL_MSO,verificationException.errorCode)
     }
 
     @Test
```

Comment:
We have now split the error for date checks of mso, can we also add respective unit tests for the same?